### DISCUSSION TITLE
Editor might benefit from having the option of a tile/control focus based cursor instead of a pixel cursor

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [Wohlstand/TheXTech#715] [Ideas]
Right now the editor cursor moves on a per pixel basis

It would speed up level development alot if the cursor worked more like a tile / control focus.

**Comment by ds-sloth** [comment]
This is a very interesting idea. I'm not sure exactly how this would work, but as a side note I'm considering adding gridlines to the level view in the editor. This may interact well with that.

**Comment by Shadowblitz16** [comment]
@ds-sloth it would basicly be like mario maker where the cursor is control / grid based instead of using something like a desktop mouse cursor.

**Comment by ds-sloth** [comment]
Okay, I will look into how some other editor engines on consoles do it. I'm definitely open to changing the design of the editor screen to work better for traditional gamepads. (As a note, it works fine with the steam controller and a touchscreen.)
